### DISCUSSION TITLE
It just works

### DISCUSSION THREAD

**Discussion by darklight9811** [egoist/tsup#497] [Show and tell]
I'm not someone who normally praises the work of others. I'm really hard to impress, but this package has done exactly that.

I believe that programming should be easy for the ones consuming things, and tsup delivers it. I've been working on a solution that fits my needs for 3 weeks, until I found this. When I get back on track on my financial situation I will definitely donate to egoist. Thanks for such an excellent package.

This really deserves more attention.

**Comment by egoist** [comment]
thanks, credits to esbuild and rollup-plugin-dts, most of the hard work are delegated to those tools.
